src/pages/iou/request/step/IOURequestStepDescription.tsx

```diff
@@ -225,8 +225,8 @@ const IOURequestStepDescriptionWithOnyx = withOnyx<IOURequestStepDescriptionProp
 })(IOURequestStepDescription);
 
 // eslint-disable-next-line rulesdir/no-negated-variables
-const IOURequestStepDescriptionWithWritableReportOrNotFound = withWritableReportOrNotFound(IOURequestStepDescriptionWithOnyx);
+const IOURequestStepDescriptionWithFullTransactionOrNotFound = withFullTransactionOrNotFound(IOURequestStepDescriptionWithOnyx);
```

Comment:
@dukenv0307 I am not entirely sure on what changes we are making here since this PR is very old and the new changes are different than the accepted proposal.

Could you let me know the changes and why we need to do this and how will it fix the issue?